src/Page/Packages.elm

```diff
@@ -148,259 +190,357 @@ update navKey msg model =
 view : Model -> Html Msg
 view model =
     Search.view { toRoute = Route.Packages, categoryName = "packages" }
-        "Search NixOS packages"
+        [ text "Search more than "
+        , strong [] [ text "80 000 packages" ]
```

Comment:
Any strategies to help ensure this number doesn't drift too far?

Reply:
Ideally this would be done as a counter that counts from 0 to a number which is fetched from the backend meanwhile. But for now this was the easier hack.